src/main/java/enums/Command.java

```diff
@@ -0,0 +1,18 @@
+package enums;
+
+public enum Command {
```

Comment:
You can consider writing JavaDoc comments for this enum class? Although it is a reach goal since you only need to do for half of the non-private classes and methods.